packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldProgrammaticSet.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Button from 'terra-button/lib/Button';
+import SearchField from '../../../SearchField';
+
+class CallbackSearchField extends React.Component {
```

Comment:
nit: CallbackSearchField -> SearchFieldProgrammaticSet / SearchFieldProgrammaticSearch

Reply:
Fixed https://github.com/cerner/terra-core/pull/2800/commits/48780c9b67df1f5c0e433435270b039142582536#diff-5b54d3bd01a4e6076cc428ee0d369f6cR9 and https://github.com/cerner/terra-core/pull/2800/commits/48780c9b67df1f5c0e433435270b039142582536#diff-5b54d3bd01a4e6076cc428ee0d369f6cR9